### ISSUE TITLE
[Fonts API] Remove all files once Font Library is merged

### ISSUE THREAD

**Issue by hellofromtonya** [WordPress/gutenberg#51820]
Related to https://github.com/WordPress/gutenberg/issues/51769 and https://github.com/WordPress/gutenberg/issues/51819

Part of https://github.com/WordPress/gutenberg/issues/41479.

Dependent upon https://github.com/WordPress/gutenberg/pull/51770/, https://github.com/WordPress/gutenberg/issues/51819, and https://github.com/WordPress/gutenberg/pull/50927 being merged into `trunk`.

## What problem does this address?

The Fonts API will be dead code once the Font Library is merged into `trunk`, as it will not load into memory. 

## What is your proposed solution?

Once Font Library, Font Face, and https://github.com/WordPress/gutenberg/issues/51819 are all merged into `trunk`, then:

1. Delete the entire `/lib/experimental/fonts-api/` folder.
2. Delete the `require` lines for each of those files in `lib/load.php`.
